parser/sql/dialect/oracle/src/main/antlr4/imports/oracle/BaseRule.g4

```diff
@@ -613,6 +612,7 @@ notOperator
 
 booleanPrimary
     : booleanPrimary IS NOT? (TRUE | FALSE | UNKNOWN | NULL)
+    | PRIOR predicate
```

Comment:
Why add this not boolean syntax to booleanPrimary?

Reply:
`booleanPrimary` refers to the grammar structure of mysql, the `booleanPrimary` concept goes beyond itself, so it can only be added here.

Reply:
> `booleanPrimary` refers to the grammar structure of mysql, the `booleanPrimary` concept goes beyond itself, so it can only be added here.

Thank you, I got it.